src-plugins/vtkDataMesh/navigators/vtkDataMeshNavigator.h

```diff
@@ -39,6 +39,8 @@ public slots:
     void updateWidgets();
 
     void enableDepthPeeling(bool);
+    void setBackgroundColor(const QString &color);
+    void setBackgroundColor(QColor color);
```

Comment:
There is small design issue here which introduces a potential bug. The function ```setBackgroundColor(const QString&)``` updates the Background Color parameter whereas ```setBackgroundColor(QColor)``` does not. If some code calls the second version directly, the parameter will not be updated as the developer might expect. Both functions should have the same result or the second one should be private as it is not a proper setter.

Reply:
These functions were initially in `vtkDataMeshInteractor` where the functions `setColor(QColor color)` and `setColor(const QString &color)` are. However, both functions update the background color: the QString one calls the QColor one.

Reply:
(sorry, you mean the parameter? I can update the name if you want)

Reply:
Yes in theory both functions should update the same state, but that's not going to be possible since you could call the second function with a color that is not in the parameters list, so the solution would be to consider the second function an internal helper function (i.e. make it private or protected).